recipes-tag/bluesky-kafka/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "0.1" %}
+
+package:
+    name: bluesky-kafka
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/nsls-ii/bluesky-kafka
+    git_rev: v{{ version }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    skip: True  # [py2k]
+
+requirements:
+    build:
+        - python
+
+    run:
+        - bluesky
+        - pykafka
```

Comment:
At ISS didn't we install `kafka-python`?  I may not have been paying close enough attention.